_episodes/l4_04_parameterized_tests.md

```diff
@@ -281,6 +282,58 @@ With this parameterised test, we were able to solve the issues that we were disc
 >
 {: .challenge}
 
+> ## Exercise 2: Multiple parameterised tests
+>
+> Suppose you have the following 3 functions that you want to test using parameterised tests:-
+>
+> 1. `int Sum(int a, int b)` as defined in the previous exercise,
+>
+> 2. `double Multiply(double a, double b)` function which multiples the two numbers and
+>
+> 3. `Power(double a, int b)` function which raises a number `a` to an integer power `b`.
```

Comment:
Minor nit: I'd write the function names as lowercase for consistency with what we've done elsewhere.

Also, `Power` is missing its return type here.